src/htmlminifier.js

```diff
@@ -978,10 +978,7 @@
             buffer.push('</' + tag + '>');
           }
           charsPrevTag = '/' + tag;
-          if (!inlineTextTags(tag)) {
```

Comment:
You need to keep this for clearing `currentChars` - try changing this to `inlineTags` instead of removing the logic.